src/group_theory/subgroup/basic.lean

```diff
@@ -1797,6 +1799,12 @@ by simp only [mem_centralizer_iff, mul_inv_eq_iff_eq_mul, one_mul]
 @[to_additive] lemma centralizer_top : centralizer ⊤ = center G :=
 set_like.ext' (set.centralizer_univ G)
 
+@[to_additive] lemma le_centralizer_iff : H ≤ K.centralizer ↔ K ≤ H.centralizer :=
+⟨λ h x hx y hy, (h hy x hx).symm, λ h x hx y hy, (h hy x hx).symm⟩
+
+@[to_additive] lemma centralizer_le (h : H ≤ K) : centralizer K ≤ centralizer H :=
```

Comment:
I think this is usually called `centralizer_antitone` or something similar (this is surely the case for monotone functions), but I am not sure.

Reply:
It's called `centralizer_le` for `submonoid`, `subalgebra`, `subsemiring`, and `subsemigroup`.

Reply:
https://leanprover-community.github.io/mathlib_docs/group_theory/submonoid/centralizer.html#submonoid.centralizer_le
https://leanprover-community.github.io/mathlib_docs/algebra/algebra/subalgebra/basic.html#subalgebra.centralizer_le
https://leanprover-community.github.io/mathlib_docs/ring_theory/subsemiring/basic.html#subsemiring.centralizer_le
https://leanprover-community.github.io/mathlib_docs/group_theory/subsemigroup/centralizer.html#subsemigroup.centralizer_le

Reply:
Ops :D

Reply:
But it's not the name I expected either. Maybe the names should be changed at some point?